### PR TITLE
Help content

### PR DESCRIPTION
**Pull request by dethos**
- Update menu order
- Updated FAQ
- Added a new static page with info to help hawkpost users. Still needs some review and design adjustments.

### CODE REVIEW

File: pages/templates/pages/help.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        <div class="col-xs-12 col-sm-9 col-md-8">
+            <p>In this page we try to explain common procedures and features that can be complex and hard to grasp for newcomers.</p>
+            <p>Some are related to Hawkpost directly but others can be related to PGP and associated tasks and tools.</p>
+            <p>Of course we can't cover all the surface or explicitly address all questions in this document, so if your doubt isn't here feel free to email us or drop by our support channels and we will try to help you as soon as possible.</p>
+
+            <p>
+                <b>Email:</b> {{support_email}}
+            </p>
+            <p>
+                <b>IRC Channel:</b> #hawkpost om freenode
```
Comment (ricardopereira) [inline pages/templates/pages/help.html:26]: Nice 😊

File: pages/templates/pages/help.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                <li>Have the required software to decrypt the messages</li>
+            </ul>
+
+            <p>If you don't know how to fulfill the first and third point, <a href="#gen-pgp-key">please check the next section</a>. Otherwise the configuration is quite simple.</p>
+
+            <img src="{% static 'images/static_key.png' %}">
+
+            <p>As you can see in the above figure, you just have to fill the <i>public key</i> field and the matching fingerprint. After this you are ready to go. We strongly recommend to also fill your personal data so it can show up in the submission page.</p>
+
+            <p>To find your key's fingerprint, you can run the follow command if you are using <i>GPG</i>:</p>
```
Comment (ricardopereira) [inline pages/templates/pages/help.html]: GPG?

File: pages/templates/pages/help.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+            <p>Since Hawkpost works only through email, the easiest way to setup a key pair is using your email client or through a compatible plugin/extension if it doesn't support PGP natively. There are lots of articles in the Internet about this topic, so it is easy to find some that match your specific setup.</p>
+
+            <p>As an example, in the next paragraphs we point to some tutorials addressing popular mail clients.</p>
+
+            <p>For webmail users, one tool that is very useful is <a href="https://www.mailvelope.com/">Mailvelope</a>. This browser extension will allow you to open the encrypted messages and also generate your keys. An introductory tutorial can be found <a href="https://www.youtube.com/watch?v=Ro3MSBS9w-A">here</a>.</p>
+
+            <p>For Thunderbird users, the common practice is to install a plugin called <a href="https://www.enigmail.net/index.php/en/">Enigmail</a>, that like Mailvelope it will allow you to decrypt Hawkpost messages and manage your keys as well. An introduction can be <a href="https://ssd.eff.org/en/module/how-use-pgp-linux">found here</a>.</p>
```
Comment (ricardopereira) [inline pages/templates/pages/help.html]: `href` is including "found" but it's not consistent with the last one that doesn't include the "found" word.

File: pages/templates/pages/help.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        </div>
+        <div class="col-xs-12 col-sm-9 col-md-8">
+            <h2 id="gen-pgp-key">How to generate a PGP key</h2>
+            <p>Hawkpost works with an encryption technology known as <a href="https://en.wikipedia.org/wiki/Pretty_Good_Privacy">PGP</a>, so to create boxes you will need to setup a PGP key pair and provide us the public part/key. If you already have one this section is not for you.</p>
+
+            <p>Since Hawkpost works only through email, the easiest way to setup a key pair is using your email client or through a compatible plugin/extension if it doesn't support PGP natively. There are lots of articles in the Internet about this topic, so it is easy to find some that match your specific setup.</p>
+
+            <p>As an example, in the next paragraphs we point to some tutorials addressing popular mail clients.</p>
```
Comment (ricardopereira) [inline pages/templates/pages/help.html]: "mail clients": replace with "email clients" to be consistent with the remaining text.

File: pages/templates/pages/help.html (hunk 2 above)
Comment (dethos) [inline pages/templates/pages/help.html]: "Gnu Privacy Guard" will changed to make it clearer.

**Comment by dethos** [comment]
@ricardopereira the issues you spotted. If the remaining look ok to you, merge at will.

**Pull request merged**